fbchat/_util.py

```diff
@@ -53,6 +53,91 @@
 ]
 
 
+class ReqUrl(object):
```

Comment:
This class was removed in #433, you've just merged your PR incorrectly ;)

Reply:
Damn it.